### PR TITLE
feat: add button to hide SAVE_CONFIG button for pending bed_mesh

### LINKED ISSUE #1066: [FR] Hide "SAVE CONFIG" button
### Requested feature:

My printer is on all the time 24/7 - Bed Mesh is part of my PRINT_START.   SAVE CONFIG is always shown and I never know what it would save, because many thing trigger the display of SAVE CONFIG.  

Bed Mesh among other things causes SAVE CONFIG button to be shown and enabled.   There is currently no way to know what values will be changed when SAVE CONFIG is pressed.   I do not ever want to use this feature because of the ambiguity of what may be changed.

This best solution is to provide a toggle button in the config that hides the SAVE CONFIG button all the time.  That way I don't need to worry about accidently pressing the button and modifying my config file.

### Solves the following problem:

_No response_

### Additional information:

_No response_

### PR DESCRIPTION
**Pull request by meteyou**
this PR fixes #1066 

screenshot deactivated:
image

screenshot activated:
image

Signed-off-by: Stefan Dej <meteyou@gmail.com>

### CODE REVIEW

File: src/locales/de.json [hunk 1]
```diff
@@ -986,6 +986,8 @@
             "GcodeThumbnails": "G-Code Vorschaubilder",
             "GcodeThumbnailsDescription": "Klicke auf die Schaltfläche um zum Leitfaden zu gelangen.",
             "Guide": "Leitfaden",
+            "HideSaveConfigButtonForBedMesh": "SAVE_CONFIG Schaltfläche ausblenden für bed_mesh",
+            "HideSaveConfigButtonForBedMeshDescription": "Die SAVE_CONFIG Schaltfläche nicht anzeigen, wenn nur ein bed_mesh ausständig ist in Klipper.",
```
Comment (th33xitus) [inline src/locales/de.json]:
```suggestion
            "HideSaveConfigButtonForBedMeshDescription": "SAVE_CONFIG nicht anzeigen, wenn nur das Speichern eines bed_mesh in Klipper aussteht.",
```

File: src/locales/de.json [hunk 2]
```diff
@@ -986,6 +986,8 @@
             "GcodeThumbnails": "G-Code Vorschaubilder",
             "GcodeThumbnailsDescription": "Klicke auf die Schaltfläche um zum Leitfaden zu gelangen.",
             "Guide": "Leitfaden",
+            "HideSaveConfigButtonForBedMesh": "SAVE_CONFIG Schaltfläche ausblenden für bed_mesh",
```
Comment (th33xitus) [inline src/locales/de.json:989]:
```suggestion
            "HideSaveConfigButtonForBedMesh": "SAVE_CONFIG Schaltfläche ausblenden bei bed_mesh Änderungen",
```

File: src/locales/en.json [hunk 3]
```diff
@@ -1000,6 +1000,8 @@
             "GcodeThumbnails": "G-Code thumbnails",
             "GcodeThumbnailsDescription": "Click on the button to get to the instructions.",
             "Guide": "Guide",
+            "HideSaveConfigButtonForBedMesh": "Hide SAVE_CONFIG button for bed_mesh",
```
Comment (th33xitus) [inline src/locales/en.json:1003]:
```suggestion
            "HideSaveConfigButtonForBedMesh": "Hide SAVE_CONFIG button for bed_mesh changes",
```

File: src/locales/en.json [hunk 4]
```diff
@@ -1000,6 +1000,8 @@
...
+            "HideSaveConfigButtonForBedMeshDescription": "Don't show the SAVE_CONFIG button, if only a bed_mesh is pending in Klipper.",
```
Comment (th33xitus) [inline src/locales/en.json:1004]:
```suggestion
            "HideSaveConfigButtonForBedMeshDescription": "Hide SAVE_CONFIG, if only bed_mesh changes are pending to be saved in Klipper.",
```

**Review by th33xitus — APPROVED** [review]

File: src/locales/de.json (hunk 1 above)
Comment (meteyou) [reply src/locales/de.json]: committed this suggestion

**Pull request merged**